### DISCUSSION TITLE
[Feature] How to get rectangle bounding box coordinates from Text Detection

### DISCUSSION THREAD

**Discussion by rama298** [open-mmlab/mmocr#1580] [General]
### What is the feature?

How to convert list of points for a each bounding box to 4 coordinates of bounding box 

Sample bounding box results:
`{
    "boundary_result": [
        [
            749.9130574464798,
            1122.8898947238922,
            748.5217530727386,
            1124.2813320159912,
            742.956535577774,
            1124.2813320159912,
            741.5652312040329,
            1125.6727693080902,
            738.7826224565506,
            1125.6727693080902,
            737.3913180828094,
            1127.0642066001892,
            733.217404961586,
            1127.0642066001892,
            731.8261005878448,
            1128.4556438922882,
            724.8695787191391,
            1128.4556438922882,
            723.478274345398,
            1129.8470811843872,
            722.0869699716568,
            1128.4556438922882,
            720.6956655979156,
            1129.8470811843872,
            719.3043612241745,
            1128.4556438922882,
            717.9130568504333,
            1129.8470811843872,
            699.8260999917984,
            1129.8470811843872,
            698.4347956180573,
            1131.2385184764862,
            677.56523001194,
            1131.2385184764862,
            676.1739256381989,
            1132.6299557685852,
            670.6087081432343,
            1132.6299557685852,
            669.2174037694931,
            1134.0213930606842,
            667.826099395752,
            1132.6299557685852,
            662.2608819007874,
            1132.6299557685852,
            660.8695775270462,
            1131.2385184764862,
            642.7826206684113,
            1131.2385184764862,
            641.3913162946701,
            1132.6299557685852,
            614.9565331935883,
            1132.6299557685852,
            613.5652288198471,
            1134.0213930606842,
            608.0000113248825,
            1134.0213930606842,
            606.6087069511414,
            1132.6299557685852,
            605.2174025774002,
            1134.0213930606842,
            603.8260982036591,
            1132.6299557685852,
            594.086967587471,
            1132.6299557685852,
            592.6956632137299,
            1134.0213930606842,
            591.3043588399887,
            1132.6299557685852,
            589.9130544662476,
            1134.0213930606842,
            588.5217500925064,
            1132.6299557685852,
            587.1304457187653,
            1134.0213930606842,
            585.7391413450241,
            1132.6299557685852,
            584.347836971283,
            1134.0213930606842,
            582.9565325975418,
            1132.6299557685852,
            581.5652282238007,
            1134.0213930606842,
            580.1739238500595,
            1132.6299557685852,
            578.7826194763184,
            1134.0213930606842,
            577.3913151025772,
            1134.0213930606842,
            576.0000107288361,
            1132.6299557685852,
            574.6087063550949,
            1134.0213930606842,
            573.2174019813538,
            1132.6299557685852,
            571.8260976076126,
            1134.0213930606842,
            570.4347932338715,
            1132.6299557685852,
            569.0434888601303,
            1132.6299557685852,
            567.6521844863892,
            1134.0213930606842,
            566.260880112648,
            1132.6299557685852,
            564.8695757389069,
            1134.0213930606842,
            563.4782713651657,
            1132.6299557685852,
            562.0869669914246,
            1132.6299557685852,
            560.6956626176834,
            1134.0213930606842,
            531.4782707691193,
            1134.0213930606842,
            530.0869663953781,
            1135.4128303527832,
            528.695662021637,
            1134.0213930606842,
            527.3043576478958,
            1134.0213930606842,
            525.9130532741547,
            1135.4128303527832,
            524.5217489004135,
            1134.0213930606842,
            523.1304445266724,
            1135.4128303527832,
            521.7391401529312,
            1135.4128303527832,
            520.3478357791901,
            1134.0213930606842,
            518.9565314054489,
            1135.4128303527832,
            517.5652270317078,
            1134.0213930606842,
            516.1739226579666,
            1135.4128303527832,
            514.7826182842255,
            1134.0213930606842,
            513.3913139104843,
            1135.4128303527832,
            512.0000095367432,
            1134.0213930606842,
            510.608705163002,
            1135.4128303527832,
            503.65218329429626,
            1135.4128303527832,
            502.2608789205551,
            1136.8042676448822,
            500.86957454681396,
            1136.8042676448822,
            499.4782701730728,
            1135.4128303527832,
            498.08696579933167,
            1136.8042676448822,
            496.6956614255905,
            1135.4128303527832,
            495.30435705184937,
            1136.8042676448822,
            493.9130526781082,
            1136.8042676448822,
            492.52174830436707,
            1135.4128303527832,
            491.1304439306259,
            1136.8042676448822,
            489.73913955688477,
            1135.4128303527832,
            480.0000089406967,
            1135.4128303527832,
            478.60870456695557,
            1136.8042676448822,
            477.2174001932144,
            1135.4128303527832,
            471.6521826982498,
            1135.4128303527832,
            470.26087832450867,
            1136.8042676448822,
            463.3043564558029,
            1136.8042676448822,
            461.91305208206177,
            1138.1957049369812,
            460.5217477083206,
            1136.8042676448822,
            459.13044333457947,
            1138.1957049369812,
            457.7391389608383,
            1136.8042676448822,
            452.1739214658737,
            1136.8042676448822,
            450.78261709213257,
            1135.4128303527832,
            449.3913127183914,
            1136.8042676448822,
            448.00000834465027,
            1135.4128303527832,
            438.2608777284622,
            1135.4128303527832,
            436.86957335472107,
            1136.8042676448822,
            435.4782689809799,
            1135.4128303527832,
            421.5652252435684,
            1135.4128303527832,
            420.17392086982727,
            1136.8042676448822,
            414.60870337486267,
            1136.8042676448822,
            413.2173990011215,
            1138.1957049369812,
            397.9130508899689,
            1138.1957049369812,
            396.5217465162277,
            1136.8042676448822,
            395.1304421424866,
            1136.8042676448822,
            393.7391377687454,
            1138.1957049369812,
            385.3913115262985,
            1138.1957049369812,
            384.0000071525574,
            1136.8042676448822,
            378.4347896575928,
            1136.8042676448822,
            377.0434852838516,
            1138.1957049369812,
            371.478267788887,
            1138.1957049369812,
            370.0869634151459,
            1136.8042676448822,
            329.7391365766525,
            1136.8042676448822,
            328.3478322029114,
            1135.4128303527832,
            326.9565278291702,
            1136.8042676448822,
            325.5652234554291,
            1135.4128303527832,
            322.7826147079468,
            1135.4128303527832,
            321.3913103342056,
            1136.8042676448822,
            320.0000059604645,
            1135.4128303527832,
            318.6087015867233,
            1136.8042676448822,
            317.2173972129822,
            1135.4128303527832,
            315.826092839241,
            1136.8042676448822,
            314.4347884654999,
            1135.4128303527832,
            313.0434840917587,
            1136.8042676448822,
            311.6521797180176,
            1135.4128303527832,
            310.2608753442764,
            1136.8042676448822,
            308.8695709705353,
            1135.4128303527832,
            307.4782665967941,
            1136.8042676448822,
            306.086962223053,
            1135.4128303527832,
            301.91304910182953,
            1135.4128303527832,
            300.5217447280884,
            1136.8042676448822,
            294.9565272331238,
            1136.8042676448822,
            293.56522285938263,
            1138.1957049369812,
            292.1739184856415,
            1138.1957049369812,
            290.78261411190033,
            1139.5871422290802,
            289.3913097381592,
            1139.5871422290802,
            283.8260922431946,
            1145.1528913974762,
            283.8260922431946,
            1146.5443286895752,
            282.43478786945343,
            1147.9357659816742,
            282.43478786945343,
            1149.3272032737732,
            281.0434834957123,
            1150.7186405658722,
            281.0434834957123,
            1156.2843897342682,
            279.65217912197113,
            1157.6758270263672,
            281.0434834957123,
            1159.0672643184662,
            281.0434834957123,
            1164.6330134868622,
            282.43478786945343,
            1166.0244507789612,
            282.43478786945343,
            1167.4158880710602,
            283.8260922431946,
            1168.8073253631592,
            283.8260922431946,
            1170.1987626552582,
            289.3913097381592,
            1175.7645118236542,
            290.78261411190033,
            1175.7645118236542,
            292.1739184856415,
            1177.1559491157532,
            293.56522285938263,
            1177.1559491157532,
            294.9565272331238,
            1178.5473864078522,
            300.5217447280884,
            1178.5473864078522,
            301.91304910182953,
            1179.9388236999512,
            306.086962223053,
            1179.9388236999512,
            307.4782665967941,
            1178.5473864078522,
            308.8695709705353,
            1179.9388236999512,
            310.2608753442764,
            1178.5473864078522,
            311.6521797180176,
            1179.9388236999512,
            313.0434840917587,
            1178.5473864078522,
            314.4347884654999,
            1179.9388236999512,
            315.826092839241,
            1178.5473864078522,
            317.2173972129822,
            1179.9388236999512,
            318.6087015867233,
            1178.5473864078522,
            320.0000059604645,
            1179.9388236999512,
            321.3913103342056,
            1178.5473864078522,
            322.7826147079468,
            1179.9388236999512,
            325.5652234554291,
            1179.9388236999512,
            326.9565278291702,
            1178.5473864078522,
            328.3478322029114,
            1179.9388236999512,
            329.7391365766525,
            1178.5473864078522,
            370.0869634151459,
            1178.5473864078522,
            371.478267788887,
            1179.9388236999512,
            377.0434852838516,
            1179.9388236999512,
            378.4347896575928,
            1178.5473864078522,
            403.4782683849335,
            1178.5473864078522,
            404.8695727586746,
            1179.9388236999512,
            416.0000077486038,
            1179.9388236999512,
            417.39131212234497,
            1181.3302609920502,
            418.7826164960861,
            1179.9388236999512,
            420.17392086982727,
            1181.3302609920502,
            421.5652252435684,
            1181.3302609920502,
            422.95652961730957,
            1179.9388236999512,
            448.00000834465027,
            1179.9388236999512,
            449.3913127183914,
            1178.5473864078522,
            450.78261709213257,
            1179.9388236999512,
            452.1739214658737,
            1178.5473864078522,
            460.5217477083206,
            1178.5473864078522,
            461.91305208206177,
            1179.9388236999512,
            467.47826957702637,
            1179.9388236999512,
            468.8695739507675,
            1181.3302609920502,
            470.26087832450867,
            1181.3302609920502,
            471.6521826982498,
            1179.9388236999512,
            477.2174001932144,
            1179.9388236999512,
            478.60870456695557,
            1178.5473864078522,
            480.0000089406967,
            1179.9388236999512,
            489.73913955688477,
            1179.9388236999512,
            491.1304439306259,
            1178.5473864078522,
            492.52174830436707,
            1179.9388236999512,
            493.9130526781082,
            1178.5473864078522,
            495.30435705184937,
            1178.5473864078522,
            496.6956614255905,
            1179.9388236999512,
            498.08696579933167,
            1178.5473864078522,
            499.4782701730728,
            1179.9388236999512,
            500.86957454681396,
            1178.5473864078522,
            502.2608789205551,
            1178.5473864078522,
            503.65218329429626,
            1179.9388236999512,
            505.0434876680374,
            1178.5473864078522,
            506.43479204177856,
            1178.5473864078522,
            507.8260964155197,
            1179.9388236999512,
            509.21740078926086,
            1178.5473864078522,
            510.608705163002,
            1179.9388236999512,
            512.0000095367432,
            1178.5473864078522,
            517.5652270317078,
            1178.5473864078522,
            518.9565314054489,
            1177.1559491157532,
            520.3478357791901,
            1178.5473864078522,
            521.7391401529312,
            1177.1559491157532,
            523.1304445266724,
            1177.1559491157532,
            524.5217489004135,
            1178.5473864078522,
            525.9130532741547,
            1177.1559491157532,
            527.3043576478958,
            1178.5473864078522,
            528.695662021637,
            1178.5473864078522,
            530.0869663953781,
            1177.1559491157532,
            531.4782707691193,
            1178.5473864078522,
            560.6956626176834,
            1178.5473864078522,
            562.0869669914246,
            1177.1559491157532,
            570.4347932338715,
            1177.1559491157532,
            571.8260976076126,
            1175.7645118236542,
            573.2174019813538,
            1177.1559491157532,
            574.6087063550949,
            1175.7645118236542,
            576.0000107288361,
            1177.1559491157532,
            577.3913151025772,
            1175.7645118236542,
            578.7826194763184,
            1175.7645118236542,
            580.1739238500595,
            1177.1559491157532,
            581.5652282238007,
            1175.7645118236542,
            582.9565325975418,
            1177.1559491157532,
            584.347836971283,
            1175.7645118236542,
            585.7391413450241,
            1177.1559491157532,
            587.1304457187653,
            1175.7645118236542,
            588.5217500925064,
            1177.1559491157532,
            589.9130544662476,
            1175.7645118236542,
            591.3043588399887,
            1177.1559491157532,
            592.6956632137299,
            1175.7645118236542,
            594.086967587471,
            1177.1559491157532,
            603.8260982036591,
            1177.1559491157532,
            605.2174025774002,
            1175.7645118236542,
            606.6087069511414,
            1177.1559491157532,
            608.0000113248825,
            1175.7645118236542,
            613.5652288198471,
            1175.7645118236542,
            614.9565331935883,
            1174.3730745315552,
            637.2174031734467,
            1174.3730745315552,
            638.6087075471878,
            1175.7645118236542,
            660.8695775270462,
            1175.7645118236542,
            662.2608819007874,
            1177.1559491157532,
            663.6521862745285,
            1175.7645118236542,
            673.3913168907166,
            1175.7645118236542,
            674.7826212644577,
            1177.1559491157532,
            676.1739256381989,
            1175.7645118236542,
            695.652186870575,
            1175.7645118236542,
            697.0434912443161,
            1174.3730745315552,
            698.4347956180573,
            1175.7645118236542,
            699.8260999917984,
            1174.3730745315552,
            717.9130568504333,
            1174.3730745315552,
            719.3043612241745,
            1172.9816372394562,
            731.8261005878448,
            1172.9816372394562,
            733.217404961586,
            1171.5901999473572,
            741.5652312040329,
            1171.5901999473572,
            742.956535577774,
            1170.1987626552582,
            752.6956661939621,
            1170.1987626552582,
            754.0869705677032,
            1171.5901999473572,
            755.4782749414444,
            1170.1987626552582,
            761.043492436409,
            1170.1987626552582,
            762.4347968101501,
            1168.8073253631592,
            765.2174055576324,
            1168.8073253631592,
            769.3913186788559,
            1164.6330134868622,
            770.782623052597,
            1164.6330134868622,
            770.782623052597,
            1163.2415761947632,
            774.9565361738205,
            1159.0672643184662,
            774.9565361738205,
            1156.2843897342682,
            776.3478405475616,
            1154.8929524421692,
            776.3478405475616,
            1149.3272032737732,
            777.7391449213028,
            1147.9357659816742,
            776.3478405475616,
            1146.5443286895752,
            776.3478405475616,
            1140.9785795211792,
            774.9565361738205,
            1139.5871422290802,
            774.9565361738205,
            1136.8042676448822,
            770.782623052597,
            1132.6299557685852,
            770.782623052597,
            1131.2385184764862,
            769.3913186788559,
            1129.8470811843872,
            768.0000143051147,
            1129.8470811843872,
            763.8261011838913,
            1125.6727693080902,
            761.043492436409,
            1125.6727693080902,
            759.6521880626678,
            1124.2813320159912,
            754.0869705677032,
            1124.2813320159912,
            752.6956661939621,
            1122.8898947238922,
            751.304361820221,
            1124.2813320159912,
            0.954223740742967
        ],
        [
            385.3913115262985,
            1033.8379080295563,
            384.0000071525574,
            1035.2293453216553,
            377.0434852838516,
            1035.2293453216553,
            375.6521809101105,
            1036.6207826137543,
            372.8695721626282,
            1036.6207826137543,
            371.478267788887,
            1038.0122199058533,
            364.5217459201813,
            1038.0122199058533,
            363.1304415464401,
            1039.4036571979523,
            361.739137172699,
            1038.0122199058533,
            360.3478327989578,
            1039.4036571979523,
            358.9565284252167,
            1038.0122199058533,
            357.5652240514755,
            1039.4036571979523,
            356.1739196777344,
            1038.0122199058533,
            354.7826153039932,
            1039.4036571979523,
            347.8260934352875,
            1039.4036571979523,
            346.4347890615463,
            1038.0122199058533,
            345.0434846878052,
            1039.4036571979523,
            343.652180314064,
            1038.0122199058533,
            342.2608759403229,
            1039.4036571979523,
            340.8695715665817,
            1038.0122199058533,
            339.4782671928406,
            1039.4036571979523,
            338.0869628190994,
            1038.0122199058533,
            336.6956584453583,
            1039.4036571979523,
            335.3043540716171,
            1038.0122199058533,
            333.913049697876,
            1039.4036571979523,
            332.5217453241348,
            1038.0122199058533,
            331.1304409503937,
            1039.4036571979523,
            329.7391365766525,
            1039.4036571979523,
            328.3478322029114,
            1038.0122199058533,
            326.9565278291702,
            1039.4036571979523,
            320.0000059604645,
            1039.4036571979523,
            318.6087015867233,
            1038.0122199058533,
            317.2173972129822,
            1039.4036571979523,
            315.826092839241,
            1038.0122199058533,
            314.4347884654999,
            1039.4036571979523,
            313.0434840917587,
            1038.0122199058533,
            311.6521797180176,
            1039.4036571979523,
            310.2608753442764,
            1038.0122199058533,
            308.8695709705353,
            1039.4036571979523,
            307.4782665967941,
            1039.4036571979523,
            306.086962223053,
            1038.0122199058533,
            304.69565784931183,
            1039.4036571979523,
            299.13044035434723,
            1039.4036571979523,
            297.7391359806061,
            1040.7950944900513,
            290.78261411190033,
            1040.7950944900513,
            289.3913097381592,
            1039.4036571979523,
            288.00000536441803,
            1040.7950944900513,
            285.21739661693573,
            1040.7950944900513,
            283.8260922431946,
            1039.4036571979523,
            282.43478786945343,
            1040.7950944900513,
            281.0434834957123,
            1040.7950944900513,
            279.65217912197113,
            1039.4036571979523,
            278.26087474823,
            1040.7950944900513,
            276.86957037448883,
            1039.4036571979523,
            275.4782660007477,
            1040.7950944900513,
            274.08696162700653,
            1039.4036571979523,
            272.6956572532654,
            1040.7950944900513,
            271.30435287952423,
            1039.4036571979523,
            269.9130485057831,
            1040.7950944900513,
            268.52174413204193,
            1039.4036571979523,
            267.1304397583008,
            1040.7950944900513,
            265.73913538455963,
            1039.4036571979523,
            264.3478310108185,
            1040.7950944900513,
            262.95652663707733,
            1039.4036571979523,
            261.5652222633362,
            1040.7950944900513,
            260.17391788959503,
            1039.4036571979523,
            258.7826135158539,
            1040.7950944900513,
            257.39130914211273,
            1040.7950944900513,
            256.0000047683716,
            1039.4036571979523,
            254.60870039463043,
            1040.7950944900513,
            253.21739602088928,
            1039.4036571979523,
            251.82609164714813,
            1040.7950944900513,
            250.43478727340698,
            1039.4036571979523,
            249.04348289966583,
            1040.7950944900513,
            247.65217852592468,
            1039.4036571979523,
            246.26087415218353,
            1039.4036571979523,
            244.86956977844238,
            1040.7950944900513,
            243.47826540470123,
            1039.4036571979523,
            240.69565665721893,
            1039.4036571979523,
            239.30435228347778,
            1038.0122199058533,
            233.73913478851318,
            1038.0122199058533,
            232.34783041477203,
            1036.6207826137543,
            225.39130854606628,
            1036.6207826137543,
            224.00000417232513,
            1035.2293453216553,
            222.60869979858398,
            1036.6207826137543,
            221.21739542484283,
            1035.2293453216553,
            219.82609105110168,
            1036.6207826137543,
            218.43478667736053,
            1036.6207826137543,
            217.04348230361938,
            1035.2293453216553,
            215.65217792987823,
            1036.6207826137543,
            210.08696043491364,
            1036.6207826137543,
            208.69565606117249,
            1038.0122199058533,
            204.52174293994904,
            1038.0122199058533,
            203.13043856620789,
            1036.6207826137543,
            201.73913419246674,
            1038.0122199058533,
            200.3478298187256,
            1036.6207826137543,
            198.95652544498444,
            1038.0122199058533,
            197.5652210712433,
            1036.6207826137543,
            196.17391669750214,
            1038.0122199058533,
            194.782612323761,
            1038.0122199058533,
            193.39130795001984,
            1036.6207826137543,
            186.4347860813141,
            1036.6207826137543,
            185.04348170757294,
            1035.2293453216553,
            183.6521773338318,
            1036.6207826137543,
            176.69565546512604,
            1036.6207826137543,
            175.3043510913849,
            1035.2293453216553,
            173.91304671764374,
            1036.6207826137543,
            168.34782922267914,
            1036.6207826137543,
            166.956524848938,
            1038.0122199058533,
            164.1739161014557,
            1038.0122199058533,
            160.00000298023224,
            1042.1865317821503,
            158.6086986064911,
            1042.1865317821503,
            158.6086986064911,
            1043.5779690742493,
            154.43478548526764,
            1047.7522809505463,
            154.43478548526764,
            1049.1437182426453,
            153.0434811115265,
            1050.5351555347443,
            153.0434811115265,
            1051.9265928268433,
            151.65217673778534,
            1053.3180301189423,
            151.65217673778534,
            1058.8837792873383,
            150.2608723640442,
            1060.2752165794373,
            151.65217673778534,
            1061.6666538715363,
            151.65217673778534,
            1067.2324030399323,
            153.0434811115265,
            1068.6238403320312,
            153.0434811115265,
            1070.0152776241302,
            154.43478548526764,
            1071.4067149162292,
            154.43478548526764,
            1072.7981522083282,
            160.00000298023224,
            1078.3639013767242,
            161.3913073539734,
            1078.3639013767242,
            162.78261172771454,
            1079.7553386688232,
            164.1739161014557,
            1079.7553386688232,
            165.56522047519684,
            1081.1467759609222,
            169.7391335964203,
            1081.1467759609222,
            171.13043797016144,
            1082.5382132530212,
            176.69565546512604,
            1082.5382132530212,
            178.0869598388672,
            1083.9296505451202,
            183.6521773338318,
            1083.9296505451202,
            185.04348170757294,
            1085.3210878372192,
            186.4347860813141,
            1083.9296505451202,
            193.39130795001984,
            1083.9296505451202,
            194.782612323761,
            1082.5382132530212,
            196.17391669750214,
            1082.5382132530212,
            197.5652210712433,
            1083.9296505451202,
            204.52174293994904,
            1083.9296505451202,
            205.91304731369019,
            1085.3210878372192,
            212.86956918239594,
            1085.3210878372192,
            214.26087355613708,
            1086.7125251293182,
            215.65217792987823,
            1085.3210878372192,
            225.39130854606628,
            1085.3210878372192,
            226.78261291980743,
            1086.7125251293182,
            228.17391729354858,
            1085.3210878372192,
            229.56522166728973,
            1086.7125251293182,
            230.95652604103088,
            1085.3210878372192,
            232.34783041477203,
            1086.7125251293182,
            233.73913478851318,
            1085.3210878372192,
            236.52174353599548,
            1085.3210878372192,
            237.91304790973663,
            1086.7125251293182,
            239.30435228347778,
            1085.3210878372192,
            240.69565665721893,
            1085.3210878372192,
            242.08696103096008,
            1086.7125251293182,
            243.47826540470123,
            1085.3210878372192,
            244.86956977844238,
            1085.3210878372192,
            246.26087415218353,
            1086.7125251293182,
            247.65217852592468,
            1086.7125251293182,
            249.04348289966583,
            1085.3210878372192,
            250.43478727340698,
            1086.7125251293182,
            251.82609164714813,
            1085.3210878372192,
            253.21739602088928,
            1086.7125251293182,
            254.60870039463043,
            1085.3210878372192,
            256.0000047683716,
            1086.7125251293182,
            257.39130914211273,
            1085.3210878372192,
            258.7826135158539,
            1085.3210878372192,
            260.17391788959503,
            1086.7125251293182,
            261.5652222633362,
            1085.3210878372192,
            262.95652663707733,
            1086.7125251293182,
            264.3478310108185,
            1085.3210878372192,
            265.73913538455963,
            1086.7125251293182,
            267.1304397583008,
            1085.3210878372192,
            268.52174413204193,
            1086.7125251293182,
            274.08696162700653,
            1086.7125251293182,
            275.4782660007477,
            1088.1039624214172,
            276.86957037448883,
            1086.7125251293182,
            278.26087474823,
            1088.1039624214172,
            279.65217912197113,
            1086.7125251293182,
            281.0434834957123,
            1086.7125251293182,
            282.43478786945343,
            1088.1039624214172,
            283.8260922431946,
            1086.7125251293182,
            292.1739184856415,
            1086.7125251293182,
            293.56522285938263,
            1088.1039624214172,
            294.9565272331238,
            1086.7125251293182,
            296.34783160686493,
            1088.1039624214172,
            297.7391359806061,
            1088.1039624214172,
            299.13044035434723,
            1086.7125251293182,
            300.5217447280884,
            1088.1039624214172,
            301.91304910182953,
            1086.7125251293182,
            303.3043534755707,
            1088.1039624214172,
            304.69565784931183,
            1086.7125251293182,
            306.086962223053,
            1088.1039624214172,
            307.4782665967941,
            1086.7125251293182,
            308.8695709705353,
            1086.7125251293182,
            310.2608753442764,
            1088.1039624214172,
            311.6521797180176,
            1086.7125251293182,
            313.0434840917587,
            1088.1039624214172,
            320.0000059604645,
            1088.1039624214172,
            321.3913103342056,
            1089.4953997135162,
            322.7826147079468,
            1088.1039624214172,
            324.1739190816879,
            1089.4953997135162,
            325.5652234554291,
            1089.4953997135162,
            326.9565278291702,
            1088.1039624214172,
            328.3478322029114,
            1088.1039624214172,
            329.7391365766525,
            1089.4953997135162,
            331.1304409503937,
            1088.1039624214172,
            332.5217453241348,
            1088.1039624214172,
            333.913049697876,
            1089.4953997135162,
            335.3043540716171,
            1088.1039624214172,
            347.8260934352875,
            1088.1039624214172,
            349.2173978090286,
            1089.4953997135162,
            350.6087021827698,
            1088.1039624214172,
            352.0000065565109,
            1089.4953997135162,
            353.3913109302521,
            1089.4953997135162,
            354.7826153039932,
            1088.1039624214172,
            361.739137172699,
            1088.1039624214172,
            363.1304415464401,
            1086.7125251293182,
            364.5217459201813,
            1088.1039624214172,
            371.478267788887,
            1088.1039624214172,
            372.8695721626282,
            1089.4953997135162,
            374.2608765363693,
            1088.1039624214172,
            375.6521809101105,
            1089.4953997135162,
            382.6087027788162,
            1089.4953997135162,
            384.0000071525574,
            1090.8868370056152,
            385.3913115262985,
            1089.4953997135162,
            392.3478333950043,
            1089.4953997135162,
            393.7391377687454,
            1090.8868370056152,
            395.1304421424866,
            1089.4953997135162,
            402.0869640111923,
            1089.4953997135162,
            403.4782683849335,
            1088.1039624214172,
            404.8695727586746,
            1088.1039624214172,
            406.26087713241577,
            1086.7125251293182,
            407.6521815061569,
            1086.7125251293182,
            409.04348587989807,
            1085.3210878372192,
            410.4347902536392,
            1085.3210878372192,
            416.0000077486038,
            1079.7553386688232,
            416.0000077486038,
            1078.3639013767242,
            417.39131212234497,
            1076.9724640846252,
            417.39131212234497,
            1075.5810267925262,
            418.7826164960861,
            1074.1895895004272,
            418.7826164960861,
            1072.7981522083282,
            420.17392086982727,
            1071.4067149162292,
            420.17392086982727,
            1064.4495284557343,
            421.5652252435684,
            1063.0580911636353,
            420.17392086982727,
            1061.6666538715363,
            420.17392086982727,
            1054.7094674110413,
            418.7826164960861,
            1053.3180301189423,
            418.7826164960861,
            1051.9265928268433,
            417.39131212234497,
            1050.5351555347443,
            417.39131212234497,
            1049.1437182426453,
            416.0000077486038,
            1047.7522809505463,
            416.0000077486038,
            1046.3608436584473,
            410.4347902536392,
            1040.7950944900513,
            409.04348587989807,
            1040.7950944900513,
            407.6521815061569,
            1039.4036571979523,
            406.26087713241577,
            1039.4036571979523,
            404.8695727586746,
            1038.0122199058533,
            403.4782683849335,
            1038.0122199058533,
            402.0869640111923,
            1036.6207826137543,
            400.6956596374512,
            1036.6207826137543,
            399.30435526371,
            1035.2293453216553,
            392.3478333950043,
            1035.2293453216553,
            390.9565290212631,
            1033.8379080295563,
            389.565224647522,
            1035.2293453216553,
            388.1739202737808,
            1033.8379080295563,
            386.7826159000397,
            1035.2293453216553,
            0.9429008881613701
        ],
  

    ],
    "filename": null
}`
![out_mcocr_warmup_0b23c1feb950a2b4e110d1aa5e9a075b_00037](https://user-images.githubusercontent.com/115778829/204475121-00d4c1e6-7eae-430a-81b9-0c51848cb0d5.png)

this is my sample img how to get rectangle bounding box

**Comment by gaotongxiao** [comment]
Check out these functions in OpenCV: `boundingRect()` and `minAreaRect()`
